app/views/play/level/PlayLevelVideoView.coffee

```diff
@@ -0,0 +1,20 @@
+RootComponent = require 'views/core/RootComponent'
+PlayLevelVideoComponent = require('./PlayLevelVideoComponent.vue').default
+utils = require 'core/utils'
+
+module.exports = class PlayLevelVideoView extends RootComponent
+  id: 'play-level-video-view'
+  template: require 'templates/base-flat'
+  VueComponent: PlayLevelVideoComponent
+
+  constructor: (options, @levelID) ->
+    @propsData = { @levelID }
+    @propsData.courseID = utils.getQueryVariable 'course' or null
```

Comment:
`getQueryVariable` 2nd param is default value:
https://github.com/codecombat/codecombat/blob/b77b425/app/core/utils.coffee#L376

Also, why explicit null instead of undefined here?

Reply:
Yes, even with `null`, it is being set as `undefined` because of the second parameter of `getQueryVariable` which is fine, so will remove explicit `null`.